Ginger/Ginger/Run/RunSetsExecutionsHistoryPage.xaml.cs

```diff
@@ -322,8 +405,8 @@ private async void LoadExecutionsHistoryData()
             if (mExecutionsHistoryList != null && mExecutionsHistoryList.Count > 0)
             {
                 IEnumerable<RunSetReport> sortedAndFilteredExecutionHistoryList = mExecutionsHistoryList
-                    .Where(report => report.RunSetExecutionStatus != eRunStatus.Automated)
-                    .OrderByDescending(item => item.StartTimeStamp);
+                   .Where(report => report.RunSetExecutionStatus != eRunStatus.Automated)
```

Comment:
Use SetContentInGrid() method instead of this code as as the same code is being executed in the function

Reply:
done